packages/react/src/date-picker/Calendar.tsx

```diff
@@ -290,17 +290,20 @@ function YearMonthSelect({
 				focusRingFor="keyboard"
 				paddingRight={1.5}
 				color="text"
-				fontSize="md"
 				fontWeight="bold"
 				css={{
-					height: '2rem',
-					borderWidth: tokens.borderWidth.sm,
-					borderStyle: 'solid',
-					borderColor: boxPalette.border,
 					appearance: 'none',
 					background: 'none',
+					borderColor: boxPalette.border,
+					borderStyle: 'solid',
+					borderWidth: tokens.borderWidth.sm,
+					fontSize: `${tokens.fontSize.xs.sm}rem`,
```

Comment:
I would like to see a `mapTokenToRem` function which is the same as `mapSpacing`, and `mapSpacing` just uses `mapTokenToRem` internally for backwards compatibility.

I feel we'll be using this a lot

Reply:
We'll also use a `pxToRem` function a lot in the future to get rid of all of the hardcoded pixel values in the codebase

Reply:
Yeah, I like this, 🙌

Reply:
Did you want to create a ticket for this?